finagle-core/src/test/scala/com/twitter/finagle/loadbalancer/TrafficDistributorTest.scala

```diff
@@ -204,7 +205,7 @@ class TrafficDistributorTest extends FunSuite {
     }
 
     locally {
-      val R = 10 * 1000
+      val R = 100 * 1000
```

Comment:
it's really bizarre that you found it didn't work.  can we try doing it with 10K again?  I'd rather not change it if we don't have to.

Reply:
I'll see if changing the seed does anything -- it's taken from a hash from a string, and there are some hash changes.